jkube-kit/build/api/src/test/java/org/eclipse/jkube/kit/build/api/auth/handler/FromConfigRegistryAuthHandlerTest.java

```diff
@@ -81,9 +84,10 @@ public void testFromPluginConfigurationPull() {
     public void testFromPluginConfigurationFailed() {
         FromConfigRegistryAuthHandler handler = new FromConfigRegistryAuthHandler(
             RegistryAuthConfig.builder().putDefaultConfig(RegistryAuth.USERNAME, "admin").build(), log);
-
-        expectedException.expect(IllegalArgumentException.class);
-        expectedException.expectMessage(containsString("password"));
+        
+        IllegalArgumentException exception= assertThrows(IllegalArgumentException.class,() -> {handler.create(RegistryAuthConfig.Kind.PUSH, null, null, s -> s); });             
```

Comment:
No need to have curly braces around single lambda statements

```suggestion
        IllegalArgumentException exception = assertThrows(IllegalArgumentException.class, () -> handler.create(RegistryAuthConfig.Kind.PUSH, null, null, s -> s));            
```

Reply:
updated